metaci/repository/utils.py

```diff
@@ -1,14 +1,18 @@
 from django.conf import settings
 from github3 import login
 
+def get_github_api(repo):
+    github = login(settings.GITHUB_USERNAME, settings.GITHUB_PASSWORD)
```

Comment:
If you use `cumulusci.core.github.get_github_api` it'll set up a `requests` session that retries on transient errors.

Reply:
I think that will work given that the service is in the db.  I'll test it out locally and push if it works.  It would mean the migration script would fail if you don't have the Github service configured in MetaCI vs using the environment variables which are required to be set per validation logic in settings.py

Reply:
I'm not following. You still pass the username and password in.

Reply:
(`get_github_api` is a helper that just gets a github3 client that autoretries, it doesn't touch the keychain)

david - we should likely move it into cumulusci.utils since it doesnt at all rely on a config/runtime/context

Reply:
the exact incantation would be 
```
github = get_github_api(settings.GITHUB_USERNAME, settings.GITHUB_PASSWORD)
return github.repository(repo.owner, repo.name)
```

it just provides some extra nicenes